packages/neuron-ui/src/widgets/AlertDialog/index.tsx

```diff
@@ -25,10 +27,17 @@ const AlertDialog = ({
   onClose?: () => void
   onOk?: () => void
   onCancel?: () => void
+  action?: Action
 }) => {
   const [t] = useTranslation()
   const dialogRef = useRef<HTMLDialogElement | null>(null)
   useDialog({ show, dialogRef, onClose: onClose || (() => {}) })
+  const actions = useMemo<('cancel' | 'ok')[]>(() => {
+    if (action) {
+      return action === 'all' ? ['cancel', 'ok'] : [action]
+    }
```

Comment:
Why is `all` mapped to `[cancel, ok]`

Reply:
`all` means show `cancel` and `ok` action buttons.